packages/teams-js/src/public/runtime.ts

```diff
@@ -236,6 +236,7 @@ interface IRuntimeV4 extends IBaseRuntime {
     readonly conversations?: {};
     readonly copilot?: {
       readonly eligibility?: {};
+      readonly customTelemetry?: {};
```

Comment:
nit: alphabetize

Reply:
@erinha  I have a question about this, does it need to be the part of runtime capability?
I mean if the hub doesn't support it, they will ignore these telemetry messages.

Reply:
done